tests/test.py

```diff
@@ -1035,7 +1053,7 @@ def get_acraserver_connection_string(self, port=None):
         """use similar to connector unix socket connection string to allow connect directory to acra by db driver"""
         if not port:
             port = self.ACRASERVER_PORT
-        return self.get_connector_connection_string(port)
+        return get_acraserver_unix_connection_string(port)
```

Comment:
hm... is it really use TLS after that? because PSQL doesn't establish TLS over unix sockets

Reply:
In case of TLS we have tcp connection string

```
def get_acraserver_unix_connection_string(port):
    if TEST_MYSQL:
        return get_tcp_connection_string(port)
    else:
        if TEST_WITH_TLS:
            return get_tcp_connection_string(port)
        else:
            return 'unix://{}/.s.PGSQL.{}'.format(PG_UNIX_HOST, port)

```